templates/inventory/dashboard/category_new.html

```diff
@@ -0,0 +1,41 @@
+{% extends "dashboard_base.html" %}
+{% load compress %}
+{% load addcss %}
+{% load crispy_forms_tags %}
+
+{% block title %}
+Kategori - Online Dashboard
+{% endblock title %}
+
+{% block styles %}
+    {{ block.super }}
+    {% compress css %}
+    {% endcompress %}
+{% endblock %}
+
+{% block page-header %}
+Kategori
+{% endblock %}
+
+{% block breadcrumbs %}
+    <li><a href="/dashboard/inventory/category/">Kategorier</a></li>
+    <li>Kategori</li>
+{% endblock %}
+
+{% block content %}
+    {%  if category.id %}
+        <div class="row">
+            <div class="col-md-12">
+                <a href="/dashboard/inventory/category/{{ category.id }}/delete/" class="btn btn-danger pull-right">
```

Comment:
Hardcoded urls? :-1: :angry: :rage:

Use django.core.urlresolvers.reverse to reverse the URLs automagically.